src/services/stringCompletions.ts

```diff
@@ -1047,7 +1125,10 @@ function getModulesForPathsPattern(
     pattern: string,
     extensionOptions: ExtensionOptions,
     isExportsWildcard: boolean,
+    isImports: boolean,
+    compilerOptions: CompilerOptions,
     host: LanguageServiceHost,
+    moduleSpecifierResolutionHost: ModuleSpecifierResolutionHost | undefined

```

Comment:
TODO: it was annoying to pass `moduleSpecifierResolutionHost` here from all of the callers but I probably should do that